app/src/components/market/sections/market_create/steps/items/create_scalar_market.tsx

```diff
@@ -125,6 +120,75 @@ export const CreateScalarMarket = (props: Props) => {
     upperBound,
   } = props
 
+  let lowerBoundError
+  let startingPointError
+  let upperBoundError
+  let startingPointNumber
+  let lowerBoundNumber
+  let upperBoundNumber
+  // if the input values aren't null the number is cast to BigNumber with 18 decimals and precision value of 2.
+  if (startingPoint !== null) {
+    startingPointNumber = formatBigNumber(startingPoint, 18, 2)
+  }
+
+  if (lowerBound !== null) {
+    lowerBoundNumber = formatBigNumber(lowerBound, 18, 2)
+  }
+
+  if (upperBound !== null) {
+    upperBoundNumber = formatBigNumber(upperBound, 18, 2)
+  }
+  //check for values to be non negative integers
+  if (startingPointNumber !== undefined && startingPointNumber < '0') {
+    startingPointError = 'Value must be greater than 0'
+  }
+
+  if (lowerBoundNumber !== undefined && lowerBoundNumber < '0') {
+    lowerBoundError = 'Value must be greater than 0'
+  }
+
+  if (upperBoundNumber !== undefined && upperBoundNumber < '0') {
+    upperBoundError = 'Value must be greater than 0'
+  }
+  //checks for numbers which don't follow the input rules: lowerBound < startingPoint < upperBound. also errors for numbers equal to each other.
+  if (
+    Number(startingPointNumber) > Number(upperBoundNumber) &&
```

Comment:
This could be `Number(startingPointNumber) >= Number(upperBoundNumber)` and then you can omit the else statement